spec/requests/api/saved_scenarios_spec.rb

```diff
@@ -94,7 +94,11 @@
       end
 
       it 'returns the saved scenario' do
-        expect(JSON.parse(response.body)).to eq(saved_scenario.as_json)
+        expect(JSON.parse(response.body)).to eq(saved_scenario.as_json.deep_transform_keys(&:to_s).merge(
+          "saved_scenario_users" => saved_scenario.saved_scenario_users.map do |user|
+            user.as_json.deep_transform_keys(&:to_s).merge("role" => user.role.to_s)
```

Comment:
The role ID is used internally. As a safeguard we are using role keywords to communicate between apps. Based on a mapping between the keywords (e.g. `:scenario_collaborator`) and the ID's, each app will know the ability of the user. Let's make sure the role ID does not accidentally come through!